native/level1double.go

```diff
@@ -312,16 +297,15 @@ func (Implementation) Daxpy(n int, alpha float64, x []float64, incX int, y []flo
 	asm.DaxpyInc(alpha, x, y, uintptr(n), uintptr(incX), uintptr(incY), uintptr(ix), uintptr(iy))
 }
 
-// Drotg gives plane rotation
-//
+// Drotg computes the plane rotation
 // _      _    _   _     _   _
 // | c  s |    | a |     | r |
 // | -s c |  * | b |   = | 0 |
 // _      _    _   _     _   _
-//
-// r = ±(a^2 + b^2)
-// c = a/r, the cosine of the plane rotation
-// s = b/r, the sine of the plane rotation
+// where
+//  r = ±(a^2 + b^2)
+//  c = a/r, the cosine of the plane rotation
+//  s = b/r, the sine of the plane rotation
 //
 // NOTE: Netlib reference implementation returns a different
 // sign for r when a or b is zero than the BLAS technical manual. Other
```

Comment:
The sentence structure should change here too. The current order makes it unnecessarily difficult to parse. This is fixed by either leading with the conditional or finishing with the conditional. I led, but the other order is fine too.